deploy/bin/import_latest_snomedct.sh

```diff
@@ -35,6 +35,6 @@ if [ $RESULT == 0 ] ; then
 else
   #  Post failure message to slack
   curl -X POST -H 'Content-type: application/json' \
-    --data '{"text":"Latest snomedct release failed to import to OpenCodelists; this is probably because no new release was found."}'\
+    --data '{"text":"Latest snomedct release failed to import to OpenCodelists; this is probably because no new release was found. tech-support please check latest SNOMED-CT release at https://www.opencodelists.org/coding-systems/latest-releases against https://isd.digital.nhs.uk/trud/users/authenticated/filters/0/categories/26/items/101/releases"}'\
```

Comment:
Could you add an entry to [the playbook][1] or [the team manual][2] that describes what someone on tech support should check. The NHS page is a login form; should I create an account?

[1]: https://docs.google.com/document/d/1Lln5N8p3hGtD3kLHJHJMWB64aTnOGpQZmqEt7ZE6d98/edit#heading=h.7wwb8hlbbnu2
[2]: https://bennettinstitute-team-manual.pages.dev/tech-group/tech-support/